plugins/inputs/nginx_plus/README.md

```diff
@@ -0,0 +1,123 @@
+# Telegraf Plugin: nginx_plus
+
+### Configuration:
+
+```
+# Read Nginx Plus' advanced status information
+[[inputs.nginx_plus]]
+  ## An array of Nginx status URIs to gather stats.
+  urls = ["http://localhost/status"]
+```
+
+### Measurements & Fields:
+
+- nginx_processes
```

Comment:
@danielnelson should these be renamed to `nginx_plus_<measurement>` ?

Reply:
Yeah, it should be easy for anyone to change the prefix with `name_override` if they prefer the shorter name.